app/views/hosts/_operating_system.html.erb

```diff
@@ -23,7 +23,9 @@
 
 <div id='root_password'>
   <%= password_f f, :root_pass, :help_inline => _("Password must be 8 characters or more"), :required => true,
-                 :unset => action_name == "edit", :keep_value => action_name == "clone"  %>
+                 :unset => action_name == "edit", :keep_value => action_name == "clone",
+                 :placeholder => (@host.root_pass.empty?) ? nil : "**********",
+                 :help_inline => (@host.root_pass_source.present?) ? _("The password is currently set by %s.") % @host.root_pass_source : nil %>
```

Comment:
I think this is needed for the source string to get translated correctly at runtime:
```suggestion
                 :help_inline => (@host.root_pass_source.present?) ? _("The password is currently set by %s.") % _(@host.root_pass_source) : nil %>
```

Reply:
The @host.root_pass_source is translated already. So, the question is, should the translation happen here or in root_pass_source?

Reply:
In the model it is marked for extraction using `N_()`, but it isn't translated yet since there is no locale context when the model is loaded. Then here we need to perform the actual translation of the string to the user locale.